version: '3.4'

services:
  db:
    image: postgis/postgis:10-3.0
    environment:
      POSTGRES_USER: django
      POSTGRES_PASSWORD: insecure!
      POSTGRES_DB: wa_pd_data
      PGDATA: /var/lib/postgresql/data/10.10
    ports:
       - "5432:5432"
    #networks: [ 'dev' ]
    volumes:
      - ./data/dbdata:/var/lib/postgresql/data

  # # TODO: incomplete! make a dockerfile eventually
  # django:
  #   image: python:3.6.10-slim
  #   environment:
  #     POSTGRES_USER: django
  #     POSTGRES_PASSWORD: insecure!
  #     POSTGRES_DB: wa_pd_data
  #     PGDATA: /var/lib/postgresql/data/10.10
  #   ports:
  #     - "8002:8000"
  #   volumes:
  #     - ./:


# networks:
#   dev:
#     driver: bridge
